src/Components/Header/Header.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import logo from '../../Assets/Images/logo.png';
```

Comment:
add empty line above this line

Reply:
small letter for Assets directory, 
should be just assets/images/logo.png 

isn't that package resolve-paths resolving paths to avoid these .../.../  
Maybe you need to add assets to configuration place of this resolve-path package to resolve that directory also